### PR TITLE
Fix live reload failing when launched from VS.

### PR DESCRIPTION
**Pull request by TimBarham**
VS generates some temporary files when saving files. Update live reload to ignore them.

### CODE REVIEW

File: src/server/utils/jsUtils.js [hunk 1]
```diff
@@ -197,10 +197,20 @@ function retryAsync(promiseFunc, maxTries, delay, iteration) {
     });
 }
 
+function stripErrorColon(err) {
```
Comment (albertinad) [inline src/server/utils/jsUtils.js]: If we need to later remove the 'error:' string, then probably we could not add it as part of the error message.

**Comment by albertinad** [comment]
Looks good to me! just a minor comment, it should not stop merge this.

File: src/server/utils/jsUtils.js (hunk 1 above)
Comment (TimBarham) [inline src/server/utils/jsUtils.js]: Hmmm... I'm not sure what you mean. This is for processing errors that come from elsewhere and often have "Error:" at the start.

Comment (albertinad) [inline src/server/utils/jsUtils.js]: You're right! I don't know what I commented this, I confused logging with throwing an error, I read above that we log an error with 'Error' but it is a log, not throwing an error. Just a problem with my brain, forget this :)

Comment (TimBarham) [inline src/server/utils/jsUtils.js]: 😄

**Comment by sgulley2** [comment]
I am installed cordova-simulator -g standalone outside visual studio because I noticed that live reload is not working when launched from vs2017 and it still does not work.  So live reload is not working for ionic 2 priojects either inside vs2017 or standalone

**Pull request merged**